includes/payment-methods/class-wc-stripe-upe-payment-method-boleto.php

```diff
@@ -0,0 +1,33 @@
+<?php
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Boleto Payment Method class extending UPE base class
+ */
+class WC_Stripe_UPE_Payment_Method_Boleto extends WC_Stripe_UPE_Payment_Method {
+
```

Comment:
@leonardola  Not related to this file but to Boleto UPE. When having an order with an amount that's not within Boleto's limits, say R$4, creating an intent fails, and the iframe for paying with a card isn't loaded. I think the shopper should still be able to pay with a card in this scenario. Is this something we could adjust?
Video with the behavior https://d.pr/v/XB6Wgw

Reply:
Chiming in again as I read the notification 😅. Probably related: #1769. Not sure if this is easy to fix 🤔

Reply:
Maybe we should tackle that in another PR?

Reply:
Glad to see you around, tbh 😄 

Gotcha. Since this seems to be a thing at the root of UPE, I think we could mention this in that issue and handle it there. I'm dropping a message on that issue and marking this conversation as resolved. Thanks Ricardo!

Reply:
@leonardola , I missed your comment when I was typing mine, sorry. Sounds good, yup. Do you want to have a separate issue for this case? Glad to create it. I was thinking of simply mentioning this on #1769 for when someone works on that.

Reply:
I already mentioned it there. So we should be good to go